client/i18n/locales/english/intro.json

```
@@ -3480,7 +3480,12 @@
           "In these lecture videos, you will learn about working with forms in React."
         ]
       },
-      "sgau": { "title": "276", "intro": [] },
+      "workshop-react-forms": {
```

Let's change this in all places it needs to be changed. 

```suggestion
      "workshop-superhero-form": {
```